clueless/views.py

```diff
@@ -1,8 +1,18 @@
 from django.http import HttpResponse
 from django.template import loader
 
+# HttpResponse functions below here 
+
 def index(request):
 	#return HttpResponse("Welcome to the world of Clue-Less!")
 	template = loader.get_template('clueless/index.html')
 	context = {}
 	return HttpResponse(template.render(context, request))
+
+def startgame(request):
```

Comment:
Only comment I have is that in the future we will want to include some method comments.  See models.py for a typical method commenting approach for python.  Otherwise this pull request looks good to me.